Mathlib/FieldTheory/LinearDisjoint.lean

```diff
@@ -144,8 +175,53 @@ theorem LinearDisjoint.symm (H : A.LinearDisjoint B) : B.LinearDisjoint A :=
 theorem linearDisjoint_comm : A.LinearDisjoint B ↔ B.LinearDisjoint A :=
   ⟨LinearDisjoint.symm, LinearDisjoint.symm⟩
 
+section
+
+variable {L' : Type*} [Field L'] [Algebra F L'] [Algebra L' E] [IsScalarTower F L' E]
+
+/-- Linear disjointness is symmetric. -/
+theorem LinearDisjoint.symm' (H : (IsScalarTower.toAlgHom F L E).fieldRange.LinearDisjoint L') :
+    (IsScalarTower.toAlgHom F L' E).fieldRange.LinearDisjoint L :=
+  Subalgebra.LinearDisjoint.symm H
+
+/-- Linear disjointness is symmetric. -/
+theorem linearDisjoint_comm' :
+    (IsScalarTower.toAlgHom F L E).fieldRange.LinearDisjoint L' ↔
+    (IsScalarTower.toAlgHom F L' E).fieldRange.LinearDisjoint L :=
+  ⟨LinearDisjoint.symm', LinearDisjoint.symm'⟩
+
+end
+
 namespace LinearDisjoint
 
+/-- Linear disjointness is preserved by algebra homomorphism. -/
+theorem map (H : A.LinearDisjoint B) {K : Type*} [Field K] [Algebra F K]
+    (f : E →ₐ[F] K) : (A.map f).LinearDisjoint (B.map f) :=
+  linearDisjoint_iff'.2 ((linearDisjoint_iff'.1 H).map f f.injective)
+
+/-- Linear disjointness is preserved by algebra homomorphism. -/
+theorem map' (H : A.LinearDisjoint L) (K : Type*) [Field K] [Algebra F K] [Algebra L K]
+    [IsScalarTower F L K] [Algebra E K] [IsScalarTower F E K] [IsScalarTower L E K] :
+    (A.map (IsScalarTower.toAlgHom F E K)).LinearDisjoint L := by
+  rw [linearDisjoint_iff] at H ⊢
+  have := H.map (IsScalarTower.toAlgHom F E K) (RingHom.injective _)
+  rw [← AlgHom.range_comp] at this
+  convert this
+  ext; exact IsScalarTower.algebraMap_apply L E K _
+
+/-- Linear disjointness is preserved by algebra homomorphism. -/
+theorem map'' {L' : Type*} [Field L'] [Algebra F L'] [Algebra L' E] [IsScalarTower F L' E]
+    (H : (IsScalarTower.toAlgHom F L E).fieldRange.LinearDisjoint L')
+    (K : Type*) [Field K] [Algebra F K] [Algebra L K] [IsScalarTower F L K]
+    [Algebra L' K] [IsScalarTower F L' K] [Algebra E K] [IsScalarTower F E K]
+    [IsScalarTower L E K] [IsScalarTower L' E K] :
+    (IsScalarTower.toAlgHom F L K).fieldRange.LinearDisjoint L' := by
```

Comment:
> This is a long statement; is it useful?

That's a good question. I'm afraid that I overengineered these three versions of `map`; what do you think? Since `IntermediateField.LinearDisjoint` is only for an intermediate field and an abstract field, so maybe `map` and `map'` is enough. (`Subalgebra` version is only for two subalgebras, so there is only a `map`.) For "maybe it's useful" reason, I added `map''`.

> Have you thought about the possibility of centering the theory around `LinearMap/AlgHom.LinearDisjoint` instead of `Submodule/Subalgebra.LinearDisjoint`? The LinearMap/AlgHom doesn't have to be injective and you'll still be able to use dot notations.

I think maybe it's not feasible; the only place using `AlgHom` is the `map` lemma. Moreover, we have `Subalgebra` and `IntermediateField` version of `LinearDisjoint`, if put them to `AlgHom` namespace then we need two separate names.